app/src/js/modules/liveeditor.js

```diff
@@ -177,6 +194,8 @@
 
             if (fieldType === 'text') {
                 field.val($(this).text());
+            } else if (fieldType === 'textarea') {
+                field.val($(this).html().replace(/&nbsp;/g, ' ').replace(/\s?<br.*?>\s?/g, '\n'));
```

Comment:
I'm thinking we should d this for ordinary text field types as well. Maybe combine this functionality with the `onblur` stuff I mentioned in my other comment so that the sanitisation isn't happening in different places.

This is a better places for sanitisation to happen because people may close the editor without first unfocusing a field that they're editing.